internal/http/service.go

```diff
@@ -22,14 +24,27 @@ func New(cfg *config.Config, database db.DB, assets Assets) (*Service, error) {
 	router.Use(WithMetrics)
 	router.Use(WithRecover)
 
-	router.Get("/", DocHandler(assets))
+	if cfg.ListIndex && cfg.EnableApi {
+		router.Get("/", ListHandler(cfg, database, assets))
```

Comment:
I would prefer to keep the `/` index page the same and have a separate page like a `/feed` instead. Maybe we can add something to the index page to link to the feed, or have a hybrid of the two being a landing page. Need to think more about what this could look like 🤔